at java.lang.Thread.dumpStack(Thread.java:1329)
at com.adventnet.nms.startnms.SessionUpdateClient.registerHandler(SessionUpdateClient.java:71)
at com.adventnet.nms.webclient.common.DynamicUpdateServlet.sessionCreated(DynamicUpdateServlet.java:100)
at org.directwebremoting.impl.DefaultScriptSessionManager.fireScriptSessionCreatedEvent(DefaultScriptSessionManager.java:392)
at org.directwebremoting.impl.DefaultScriptSessionManager.getScriptSession(DefaultScriptSessionManager.java:109)
at org.directwebremoting.impl.DefaultWebContext.checkPageInformation(DefaultWebContext.java:87)
at org.directwebremoting.dwrp.BaseCallHandler.handle(BaseCallHandler.java:97)
at org.directwebremoting.servlet.UrlProcessor.handle(UrlProcessor.java:118)
at org.directwebremoting.servlet.DwrServlet.doPost(DwrServlet.java:144)
at javax.servlet.http.HttpServlet.service(HttpServlet.java:709)
---
Full trace:

[31 May 2017 16:07:08:547] SYS_ERR: java.lang.Exception: Stack trace
[31 May 2017 16:07:08:547] SYS_ERR: 	at java.lang.Thread.dumpStack(Thread.java:1329)
[31 May 2017 16:07:08:547] SYS_ERR: 	at com.adventnet.nms.startnms.SessionUpdateClient.registerHandler(SessionUpdateClient.java:71)
[31 May 2017 16:07:08:548] SYS_ERR: 	at com.adventnet.nms.webclient.common.DynamicUpdateServlet.sessionCreated(DynamicUpdateServlet.java:100)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.directwebremoting.impl.DefaultScriptSessionManager.fireScriptSessionCreatedEvent(DefaultScriptSessionManager.java:392)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.directwebremoting.impl.DefaultScriptSessionManager.getScriptSession(DefaultScriptSessionManager.java:109)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.directwebremoting.impl.DefaultWebContext.checkPageInformation(DefaultWebContext.java:87)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.directwebremoting.dwrp.BaseCallHandler.handle(BaseCallHandler.java:97)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.directwebremoting.servlet.UrlProcessor.handle(UrlProcessor.java:118)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.directwebremoting.servlet.DwrServlet.doPost(DwrServlet.java:144)
[31 May 2017 16:07:08:548] SYS_ERR: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:709)
[31 May 2017 16:07:08:548] SYS_ERR: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:802)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:252)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:173)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:213)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:178)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:432)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:126)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:105)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:541)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:107)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:148)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:869)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.coyote.http11.Http11BaseProtocol$Http11ConnectionHandler.processConnection(Http11BaseProtocol.java:664)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.tomcat.util.net.PoolTcpEndpoint.processSocket(PoolTcpEndpoint.java:527)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.tomcat.util.net.LeaderFollowerWorkerThread.runIt(LeaderFollowerWorkerThread.java:80)
[31 May 2017 16:07:08:548] SYS_ERR: 	at org.apache.tomcat.util.threads.ThreadPool$ControlRunnable.run(ThreadPool.java:684)
[31 May 2017 16:07:08:548] SYS_ERR: 	at java.lang.Thread.run(Thread.java:745)
[31 May 2017 16:07:08:548] SYS_ERR: Registering user com.adventnet.nms.webclient.fault.alarm.AlarmUpdateHandler for module alert
